src/msg/async/net_handler.cc

```diff
@@ -126,8 +129,6 @@ int NetHandler::generic_connect(const entity_addr_t& addr, bool nonblock)
     return -errno;
   }
 
-  set_socket_options(s);
-
```

Comment:
@yangruifeng, thanks for testing and offering man page. so you are right! we need to set `RCVBUF` **before** listen(2).